CoreFoundation/Base.subproj/CFInternal.h

```diff
@@ -97,9 +97,9 @@ CF_EXTERN_C_BEGIN
 
 #if TARGET_OS_MAC || TARGET_OS_LINUX || TARGET_OS_BSD
 
-#if !TARGET_OS_CYGWIN && !defined(__linux__)
+#if !TARGET_OS_CYGWIN && !defined(__linux__) && !defined(__OpenBSD__)
```

Comment:
Not related to your change, but this is confusing.  I assume that the condition in conjunction with L98 means that this is for Android, macOS, and *BSD but not OpenBSD.  Would you mind rewriting this as such: `#if defined(ANDROID) || (TARGET_OS_BSD && !defined(__OpenBSD__)) || TARGET_OS_MAC`

Reply:
Done. (Used TARGET_OS_ANDROID instead of defined(ANDROID) and reordered the clauses.)